coalahtml/coala_html.py

```diff
@@ -35,7 +35,7 @@ def main():
 
     dir_path = create_dir(os.path.abspath(args.dir))
 
-    if len(list(os.walk(dir_path))) > 0:
+    if len(list(os.walk(dir_path))) > 0:  # pragma: no cover
```

Comment:
So I can think of something like:
* Check if the directories differ - True or False
* Have a `--reload` cli argument that if passed, would update the app provided directories differ  otherwise keep the files intact.
@Uran198  Seems sane?